package.json

```diff
@@ -31,12 +31,14 @@
     "clone": "^2.1.2",
     "cors": "^2.8.5",
     "express": "^4.17.1",
+    "handy-redis": "^1.8.3",
```

Comment:
Instead of redis and handy-redis, ioredis might be worth looking into. Comes with support for promises right out of the box, has good performance, and has support for Lua script.